### PR TITLE
feat(core): Introduce ERS mode, ability to connect to remote ERS

### PR DESCRIPTION
**Pull request by elizabethhealy**
### Proposed Changes

* ERS mode to run an ERS on its own
* Ability to connect to a remote ers:
  * sdk config updated to include connection params: endpoint, plaintext, insecure

End result for configs: if running in core more you need to provide a sdk_config since an ERS connection is needed for authorization, ability to run in entityresolution mode, ability to configure core and ers connections with sdk_config (plaintext, insecure, endnpoint)

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation

### CODE REVIEW

File: .github/workflows/checks.yaml [hunk 1]
```diff
@@ -170,7 +170,7 @@ jobs:
           push: false
 
   platform-xtest:
-    uses: opentdf/tests/.github/workflows/xtest.yml@main
```
Comment (elizabethhealy) [inline .github/workflows/checks.yaml:173]: will remove, for testing of updated "start additional kas" action

**Comment by elizabethhealy** [comment]
successful xtest run https://github.com/opentdf/platform/actions/runs/12015513478/job/33493730789

**Comment by elizabethhealy** [comment]
xtest is failing because it call a reusable workflow in the repo on the main branch. when running against an xtest branch that calls the reusable workflow from this branch (which has been updated) it passes https://github.com/opentdf/platform/actions/runs/12015513478/job/33493730789

**Review by pflynn-virtru — APPROVED** [review]

File: docs/configuration.md [hunk 2]
```diff
@@ -34,8 +34,12 @@ Root level key `sdk_config`
 
 | Field    | Description  | Default  | Environment Variable |
 | -------- | -------------| -------- | -------------------- |
-| `endpoint` | The core platform endpoint to connect to |  | OPENTDF_SDK_CONFIG_ENDPOINT |
-| `plaintext` | Use a plaintext grpc connection | `false` | OPENTDF_SDK_CONFIG_PLAINTEXT |
+| `core.endpoint` | The core platform endpoint to connect to |  | OPENTDF_SDK_CONFIG_ENDPOINT |
+| `core.plaintext` | Use a plaintext grpc connection | `false` | OPENTDF_SDK_CONFIG_PLAINTEXT |
+| `core.insecure` | Use an insecure tls connection | `false` |  |
+| `entityresoluytion.endpoint` | The entityresolution endpoint to connect to |  |  |
```
Comment (dmihalcik-virtru) [inline docs/configuration.md:40]:
```suggestion
| `entityresolution.endpoint` | The entityresolution endpoint to connect to |  |  |
```

File: opentdf-ers-mode.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
+mode: entityresolution
```
Comment (dmihalcik-virtru) [inline opentdf-ers-mode.yaml:1]: Maybe add comments here and atop other `-mode.yaml` files about what they are showing off and how to use them, or maybe a pithy comment about why it exists

File: service/internal/config/config.go [hunk 4]
```diff
@@ -137,8 +148,12 @@ func (c *Config) LogValue() slog.Value {
 
 func (c SDKConfig) LogValue() slog.Value {
 	return slog.GroupValue(
-		slog.String("endpoint", c.Endpoint),
-		slog.Bool("plaintext", c.Plaintext),
+		slog.String("core endpoint", c.CorePlatformConnection.Endpoint),
```
Comment (dmihalcik-virtru) [inline service/internal/config/config.go:151]: maybe make these a hierarchichal group instead (e.g. core.endpoint, ers.endpoint). At least no spaces

**Review by strantalis — APPROVED** [review]

**Pull request merged**